tests/k6/tests/enduser/dialogSystemLabelLog.js

```
@@ -0,0 +1,60 @@
+    let dialogId;
+    let dialog;
+    describe('Arrange: Create a dialog to test against', () => {
+        let d = dialogToInsert();
+        let r = postSO("dialogs", d);
+        expectStatusFor(r).to.equal(201);
+        dialogId = r.json();
+        dialog = d;
+    });
+
+    describe('Update label as enduser', () => {
+        let body = {
```

Biome

[error] This let declares a variable that is only assigned once.

'body' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)